AUTHORS

```diff
@@ -1377,3 +1377,4 @@ Mathis Cros <mathis.cros@telecom-paris.fr>
 Arnav Mummineni <45217840+RCoder01@users.noreply.github.com>
 Thangaraju Sibiraj <85477603+t-sibiraj@users.noreply.github.com>
 KJaybhaye <krushnajaybhaye01@gmail.com>
+cp50 <chrispaul68002@gmail.com>
```

Comment:
Delete this line. You should not add yourself to the AUTHORS file. That will be done later.